### ISSUE TITLE
Trouble with 5-5-1

### ISSUE THREAD

**Issue by dawncomer** [dawncomer/open-sdg-site-starter#23]
Hi @brockfanning I'm having trouble with the [representation of the 5-5-1 data set](https://dawncomer.github.io/open-sdg-site-starter/5-5-1/).
The initial graph that shows on the page, spans the years 2009-2017 (Elected Offical (All), #of seats held by Women (see first screenshot below).

However, when I try to choose the dis-aggregated data (Elected roles) the chart only shows 2017 data, along the vertical axis, although there is data across a span of years (see second screenshot below).  Is this happening due to the number of Zeros in the [dataset](https://github.com/dawncomer/open-sdg-data-starter/blob/develop/data/indicator_5-5-1.csv)?   Thanks!

![5-5-1 trouble](https://user-images.githubusercontent.com/46500838/53536970-342b6580-3abd-11e9-85bc-8a5db4769395.PNG)

![5-5-1 trouble disaggregate](https://user-images.githubusercontent.com/46500838/53537069-91271b80-3abd-11e9-9cb3-c363648149cd.PNG)

**Comment by brockfanning** [comment]
@dawncomer Really interesting bug. It does seem like it's related to the zeroes, since the two values that give the problem, "Mayoral" and "City Atty", are both all zeroes. But regardless it's a bug that we should try to fix. I'll start a thread about it in the open-sdg issue queue.

**Comment by dawncomer** [comment]
Hi @brockfanning this [UK target](https://sustainabledevelopment-uk.github.io/8-3-1/) had a similar disaggregation across a span of years (with some year w/no data) as I was working to illustrate. I reviewed their source data and noted that fields with Zeros were not included.  I replicated this approach and [5-5-1 is displaying much better](https://dawncomer.github.io/open-sdg-site-starter/5-5-1/) when the disaggregated fields are selected.  #lessonlearned  thanks!
